hydra-access-controls/app/models/hydra/access_controls/permission.rb

```diff
@@ -5,9 +5,10 @@ module Hydra::AccessControls
   class Permission < AccessControlList
     has_many :admin_policies, inverse_of: :default_permissions, class_name: 'Hydra::AdminPolicy'
 
+    # We assume both args[:name] and args[:type] to be able to respond to #to_s.
```

Comment:
The YARD way to say this would be:
```ruby
# @param [Hash] args
# @option args [#to_s] :name something about the name
# @option args [#to_s] :type uhh... some kinda type
# @option args [#to_s] :access description of access
```